plugins/services/src/js/reducers/serviceForm/__tests__/Container-test.js

```diff
@@ -40,7 +40,7 @@ describe('Container', function () {
             image: 'foo',
             forcePullImage: null,
             privileged: null,
-            network: undefined,
```

Comment:
Added and actually fixed the tests. Unfortunately a lot of the container tests were not executed.